packages/earth-admin/src/styles/styles.scss

```diff
@@ -32,6 +32,18 @@ body {
   scrollbar-color: $marapp-gray-7 $marapp-gray-8;
 }
 
+input,
```

Comment:
Copied input and button reset to fix a small css problem in the user menu on admin